lms/static/js/learner_dashboard/models/course_card_model.js

```diff
@@ -148,21 +150,25 @@
                         endDate = new Date(end);
 
                     if (pacingType === 'self_paced') {
-                        dateString = 'Self-paced';
-                        if (start && startDate > now) {
-                            dateString += ' - Starts ' + start;
+                        dateString = gettext('(Self-paced) ');
+                        if (start) {
+                            dateString += startDate > now ?
+                                StringUtils.interpolate(gettext('Starts {start}'), {start: start}) :
+                                StringUtils.interpolate(gettext('Started {start}'), {start: start});
                         } else if (end && endDate > now) {
-                            dateString += ' - Ends ' + end;
+                            dateString += StringUtils.interpolate(gettext('Ends {end}'), {end: end});
                         } else if (end && endDate < now) {
```

Comment:
What if `end === now`?

Reply:
It's a nearly impossible situation since it uses milliseconds